Example/Podfile

```diff
@@ -1,12 +1,15 @@
-source 'https://github.com/CocoaPods/Specs.git'
-use_frameworks!
```

Comment:
-source 'https://github.com/CocoaPods/Specs.git'
I think above line not required because pod install command was not worked with that pattern as it was old pattern . So i created the new pod file with slightly new pattern .

use_frameworks!
I think it will be required . I missed that line while creating new pod file . 
If that's not present, we won't be able to integrate dependencies, if we depend on a pod which includes Swift source code.